.githooks/run-hooks

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+hook="${0##*/}"
+dotd="${0}.d"
+orig="${0%/*}/../.git/hooks/$hook"
+
+exec 1>&2
+
+for h in "$dotd"/???*; do
+    case "$h" in
+        *~|*.swp)
+            continue
+            ;;
+        [0-9]*-*)
+            ;;
```

Comment:
What is the significance of this?

Reply:
Hmm, in its current form it's merely documenting/suggesting how the hook-lets should be named with/for explicit ordering. But perhaps we should explicitly ignore the rest... which I suspect might have been my original intention at some point.

Added a default case for that effect and updated the PR.